.github/workflows/build-standalone.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  release:
+    types: [created]
+
+name: Build Standalone CLI
+
+jobs:
+  build_cli:
+    name: Build CLI on MacOS
+    runs-on: macos-latest
+    env:
+      APPLE_APP_STORE_CONNECT_KEY_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_KEY_ID }}
+      APPLE_APP_STORE_CONNECT_ISSUER_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_ISSUER_ID }}
+      APPLE_CODESIGN_CERT_ID: "F9F7EEE981D7A60F83EBEC36AD675448CA0AB41B"
```

Comment:
Make this a repo variable so it can be used for multiple workflows.

Repo vars are similar to repo secrets, just not secret 😆 and can be referenced like this:
```
${{ vars. APPLE_CODESIGN_CERT_ID }}
```

If the other two IDs aren't secrets, they should be moved to repo vars as well.

Reply:
@aceat64 to make a follow-up ticket for this work at a later time